common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -287,6 +287,8 @@
   "feature.wynntils.itemHighlight.zeroStarIngredientHighlightColor.name": "0 Star Ingredient Highlight Color",
   "feature.wynntils.itemHighlight.zeroStarIngredientHighlightEnabled.description": "Enable highlighting of 0 star ingredient items",
   "feature.wynntils.itemHighlight.zeroStarIngredientHighlightEnabled.name": "0 Star Ingredient Highlight",
+  "feature.wynntils.itemLock.allowClickOnEmeraldPouchInBlockingMode.description": "If Block ALl Actions on Locked Items is enabled, should clicking on the Emerald Pouch be allowed?",
```

capitalization problem

```suggestion
  "feature.wynntils.itemLock.allowClickOnEmeraldPouchInBlockingMode.description": "If Block All Actions on Locked Items is enabled, should clicking on the Emerald Pouch be allowed?",
```